doc/changelog/07-vernac-commands-and-options/15274-tacopt.rst

```diff
@@ -0,0 +1,8 @@
+- **Changed:** commands setting tactic options (currently
+  :opt:`Firstorder Solver` and :cmd:`Obligation Tactic`, as well as any
+  defined by third party plugins) now support :attr:`export` locality.
```

Comment:
This is somewhere between vague and circular: how do you know which commands defined by third party plugins are commands that set tactic options?  Perhaps you mean options/commands that have a value in the form `@ltac_expr` as the argument? Looks like `Obligation Tactic` should have been an option.  Is that worth cleaning up later?

"commands setting" is a bit loose since it also includes options.  Maybe "commands and options setting ..." is better?

Reply:
It's impossible to know without looking at the plugin's source.

Reply:
Then how about referring to `@ltac_expr` if that's what matters?

Reply:
It is not a grammar thing

Reply:
Agreed, not a grammar thing.  IIUC "tactic options" would be clearer as "options whose value is a tactic".  That's what I was trying to get at with `@ltac_expr`.

Reply:
It's really something each plugin using this system needs to say in their own doc, see paramcoq overlay